sw/device/tests/aon_timer_smoketest.c

```diff
@@ -16,6 +17,10 @@
 
 OTTF_DEFINE_TEST_CONFIG();
 
+static dif_aon_timer_t aon;
+static dif_rv_core_ibex_t rv_core_ibex;
+static bool wdog_fired = false;
```

Comment:
Should this be volatile since it crosses the ISR boundary?

Reply:
I guess it's only read once so I'm not sure if volatile is needed, but it can't hurt

Reply:
Hum, I think it does not technically need to but you are right that it doesn't hurt, in case the compiler would like to do fancy optimization.